lib/Reporter.js

```diff
@@ -1,16 +1,34 @@
+const chalk = require('chalk');
+
 module.exports = class Reporter {
   constructor (options = {}) {
     this.options = options;
   }
-  start(){}
-  result(host, result) { }
-  end(){}
+  start() {}
+  result(host, result) {}
+  end() {}
+
+  get isUnanimous() {
+    if (this.results.length === 1) {
+      return true;
+    }
+    // Get the result string of the first result entry
+    let first = this.results[0][1];
 
-  static coalesceInto(table, host, result, resultsMap /*{result:[...hosts]}*/, sep = "\n") {
+    // Compare the first result string to all result strings
+    for (let [ host, result ] of this.results) {
+      if (result !== first) {
+        return false;
+      }
+    }
+    return true;
+  }
+
+  static coalesceInto(records, host, result, resultsMap /*{result:[...hosts]}*/, sep = "\n") {
     let found = false;
 
-    for (let row of table) {
-      if (row[1] === result) {
+    for (let record of records) {
```

Comment:
This might not always be a "table", and the default reporter shouldn't be talking about tables anyway ;)